rswag-specs/spec/rswag/specs/request_factory_spec.rb

```diff
@@ -589,38 +557,12 @@ module Specs
               expect(request[:path]).to eq('/blogs?q1=foo')
             end
           end
-
-          context 'openapi 3.0.1 upgrade notice' do
-            let(:swagger_doc) { { openapi: '3.0.1' } }
-            before do
-              allow(ActiveSupport::Deprecation).to receive(:warn)
-              swagger_doc[:parameters] = { q1: { name: 'q1', in: :query, type: :string } }
-              metadata[:operation][:parameters] = [{ '$ref' => '#/parameters/q1' }]
-              allow(example).to receive(:q1).and_return('foo')
-            end
-
-            it 'warns the user to upgrade' do
-              expect(request[:path]).to eq('/blogs?q1=foo')
-              expect(ActiveSupport::Deprecation).to have_received(:warn)
-                .with('Rswag::Specs: WARNING: #/parameters/ refs are replaced in OpenAPI3! Rename to #/components/parameters/')
-              expect(ActiveSupport::Deprecation).to have_received(:warn)
-                .with('Rswag::Specs: WARNING: parameters is replaced in OpenAPI3! Rename to components/parameters (in swagger_helper.rb)')
-            end
-          end
         end
 
         context 'base path' do
-          context 'openapi 2.0' do
-            before { swagger_doc[:basePath] = '/api' }
-
-            it 'prepends to the path' do
-              expect(request[:path]).to eq('/api/blogs')
-            end
-          end
-
           context 'openapi 3.0' do
```

Comment:
I'm sure sure this `context` block is still necessary if there's just the one test within `context 'base path'` 🤔 
Thoughts?

Reply:
True. Removing it, since now that only OpenAPI v3 is supported, it does not make sense to separate the context.

Thanks!